lgtm.yml

```diff
@@ -0,0 +1,19 @@
+##########################################################################################
+# Customize file classifications.                                                        #
+# Results from files under any classifier will be excluded from LGTM                     #
+# statistics.                                                                            #
+##########################################################################################
+
+##########################################################################################
+# Use the `path_classifiers` block to define changes to the default classification of    #
+# files.                                                                                 #
+##########################################################################################
+
+extraction:
+  # Define settings for Java analysis
+  ####################################
+  java:
+    # The `index` step extracts information from the files in the codebase.
+    index:
+      # Specify the Java version required to build the project.
+      java_version: 1.8
```

Comment:
According to the docs, that should be the default value. So this change shouldn't be required anyway. 

What error were you receiving before?

![image](https://user-images.githubusercontent.com/3900123/98718035-a5cc4580-236c-11eb-8aff-aa2b261e3d4f.png)

Reply:
Ultimately we're running into this bug as @lbalmaceda discovered: https://github.com/gradle/gradle/issues/15049. I've removed the toolchain config and the lgtm.yml config.